Main/src/main/java/com/main/acad/servlet/ChildServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.main.acad.servlet;
+
+import com.main.acad.dao.ChapterDao;
+import com.main.acad.dao.Dao;
+import com.main.acad.serializator.MyJsonSerializer;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+public class ChildServlet extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse response) throws ServletException, IOException {
+        Dao d = new ChapterDao();
+        List child = null;
+        try {
+            child = d.listChildren(3);// як можна підставити значення того чи іншого заголовка??
```

Comment:
try to get firstly the list of the parent chapter and then you can get children for each of them as you will have their ids